src/components/RoomMainUi.tsx

```diff
@@ -0,0 +1,16 @@
+// HOST ROOM
+'use client'
+
+import { socketRoomEmitter } from '@/services/socket'
+
+import StoryPoints from '@/components/StoryPoints'
+import CardsContainer from '@/components/CardsContainer'
+
+export default function RoomMainUi() {
```

Comment:
suggestion: Calling this `HostRoom` :)

Reply:
It's not actually the Host Room. It's the Room UI for host AND user. It is the user cards and story point buttons and is common to both host and user. 

The host then gets buttons above this component, and the settings button.